### DISCUSSION TITLE
Some applications are not shown in on/off modules

### DISCUSSION THREAD

**Discussion by Vendict** [LSPosed/LSPosed#973] [Q&A] [answered]
Dont show some applications in on/off modules.
Applications? for example:
PokemonGO, Ingress

**Comment by yujincheng08** [comment]
I cannot understand your english.

**Comment by Vendict** [comment]
On switch on/off module I cant select some applications, because they are not on the list

**Comment by yujincheng08** [comment, marked as answer]
disable hide from the top right three dots button -> hide

**Comment by Vendict** [comment]
thank you, sorry
